bin/bridge-client/src/constants.rs

```diff
@@ -4,5 +4,6 @@ pub(super) const DEFAULT_RPC_PORT: u32 = 4781; // first 4 digits in the sha256 o
 
 pub(super) const DEFAULT_RPC_HOST: &str = "127.0.0.1";
 
+pub(super) const DEFAULT_DUTY_TIMEOUT_DURATION: u64 = 600;
```

Comment:
Units?

I can infer that this is seconds, so it should be named `DEFAULT_DUTY_TIMEOUT_SEC`.  We can leave out the `DURATION` here since it's a timeout and it should be obvious from context.